### DISCUSSION TITLE
llvm version

### DISCUSSION THREAD

**Discussion by yuqisun** [tancheng/CGRA-Mapper#32] [Q&A]
Have to use llvm-12 (ubuntu 20.04) right? Latest ubuntu 24.04 LTS llvm version is 18.

**Comment by tancheng** [comment]
u can install llvm-12 in any ubuntu version i think?

**Comment by tancheng** [comment]
We can also try whether llvm-18 works or not :-)

**Comment by yuqisun** [comment]
Tried some solutions to install version 12, doesn't work, re-installed 20.04, let me finish GUI test first. Then try with OS and LLVM upgrade.
